src/interface.jl

```diff
@@ -29,7 +29,6 @@ end
 # 2. integrate(solver, times) -> iterator
 # 3. integrate(callback, solver, times) -> vector of states with callback applied
 
-integrate(solver::AbstractDPSolver{T}, time::T) where {T <: Real} = core_integrator(solver, time)
 integrate(solver::AbstractDPSolver{T}, times::AbstractVector{T}) where {T <: Real} = DP5Iterator(solver, times)
```

Comment:
This naming seems a bit off considering DP5Iterator can now consist of a DP5 *or* DP8 Solver. Perhaps it would be best to rename it to `DPIterator` just to drive the point home that it works for both solvers.